db/schema.rb

```diff
@@ -10,52 +10,52 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_08_17_055131) do
+ActiveRecord::Schema[7.0].define(version: 2023_08_17_101227) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "comments", force: :cascade do |t|
-    t.bigint "user_id_id", null: false
-    t.bigint "post_id_id", null: false
+    t.bigint "user_id", null: false
```

Comment:
- [ ] **[OPTIONAL]** This is beyond the scope of this project, as it was handled in a previous project, but I would strongly suggest you rename this table column to `author_id`, instead of `user_id` to match the associations.